test/lit/passes/roundtrip-gc.wast

```diff
@@ -21,20 +21,24 @@
  (func $test
   (call $help
    (struct.new_default ${i32})
-   ;; Stack IR optimizations can remove this block, leaving a nop in an odd
+   ;; Stack IR optimizations can remove this block, leaving a call in an odd
    ;; "stacky" location. On load, we will use a local to work around that. It
    ;; is fine for the local to be non-nullable since the get is later in that
    ;; same block.
    (block $block (result i32)
-    (nop)
+    (call $other)
```

Comment:
This change avoids the test becoming trivial (the new opt removes a nop, after which a lot more optimizations kick in that we don't want here).